iModelCore/ecobjects/test/NonPublished/SchemaComparerTests.cpp

```diff
@@ -1121,6 +1121,54 @@ TEST_F(SchemaCompareTest, CompareECClassIdentical)
 //----------------------------------------------------------------------------------------
 // @bsimethod
 //---------------+---------------+---------------+---------------+---------------+--------
+TEST_F(SchemaComparerXmlTests, CompareSchemasWithWrongPropertyTags)
+    {
+    //Test created for 3.1 version of schema specifically 
+    //to observe defaulting behavior for and when the property tags are wrong

```

Comment:
I've seen this in a few other places but you use 'tag' when I think you mean property type: https://www.itwinjs.org/bis/ec/ec-property/#ecproperty